src/init1.cc

```diff
@@ -1956,12 +1916,9 @@ errr init_player_info_txt(FILE *fp)
 			if (8 != sscanf(buf + 4, "%d:%d:%d:%d:%d:%d:%d:%d",
 			                &s[0], &s[1], &s[2], &s[3], &s[4], &s[5], &s[6], &s[7])) return (1);
 
-			rp_ptr->r_dis = s[0];
```

Comment:
This needs a follow-up at some point to actually remove these fields from the file entirely.